docs/quickstart.md

```diff
@@ -40,6 +40,7 @@ The template will deploy in your Azure subscription the Following resources:
 - [Redis Cache](https://azure.microsoft.com/en-us/services/cache/)
 - [Application Insights](https://docs.microsoft.com/en-us/azure/azure-monitor/app/app-insights-overview)
 - [Log Analytics](https://docs.microsoft.com/en-us/azure/azure-monitor/logs/log-analytics-overview)
+- [App Service](https://docs.microsoft.com/en-us/azure/app-service/overview) (when opted in to use standalone discovery service)
```

Comment:
```suggestion
- [App Service](https://docs.microsoft.com/en-us/azure/app-service/overview) (when opted in to use standalone [discovery service](#lns-discovery))
```
There is a link already further down under the instructions, but this is the first time the discovery is mentioned